components/i2c/stemma_soil/definition.json

```diff
@@ -6,6 +6,10 @@
       "displayName": "Ambient Temperature",
       "sensorType": "ambient-temp"
     },
+    {
+      "displayName": "Ambient Temperature (Fahrenheit)",
+      "sensorType": "ambient-temp-fahrenheit"
+      },
```

Comment:
Just noticed this, you don't need to type out the full types if they don't change from what is in sensors.json. This entire stemma "subcomponents" property can be:
```
  "subcomponents": [
    "ambient-temp",
    "ambient-temp-fahrenheit",
    {
      "displayName": "Capacitive Sensor",
      "sensorType": "raw"
    }
  ]
```

...because only the "raw" type overrides anything in the defaults. Does that make sense?

Reply:
This definition was added by a contributor, will do tho.